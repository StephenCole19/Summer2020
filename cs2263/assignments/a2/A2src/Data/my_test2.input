q
u 7
o
x

