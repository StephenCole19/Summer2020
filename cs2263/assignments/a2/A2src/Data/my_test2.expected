invalid instruction q
7
7
Successfully executed 2 instructions
0 elements
